docs/guides/ai-transpiler-passes.mdx

```diff
@@ -5,11 +5,10 @@ description: What are the AI transpiler passes and how to use them
 
 # AI transpiler passes
 ​
-The AI-powered transpiler passes are experimental passes that work as a drop-in replacement of "traditional" Qiskit passes for some transpiling tasks. They often produce better results than existing heuristic algorithms (such as lower depth and CNOT count), but are also much faster than optimization algorithms such as Boolean satisfiability solvers. The AI transpiler passes run on the cloud and are available to IBM Quantum&trade; Premium Plan users.
+The AI-powered transpiler passes are experimental passes that work as a drop-in replacement of "traditional" Qiskit passes for some transpiling tasks. They often produce better results than existing heuristic algorithms (such as lower depth and CNOT count), but are also much faster than optimization algorithms such as Boolean satisfiability solvers. The AI transpiler passes can run on your local environment or on the cloud using the Qiskit Transpiler Service if you are part of the IBM Quantum&trade; Premium Plan.
```

Comment:
Can all AI transpiler passes run locally, or just the routing pass? Do they have the same exact capabilities?

Reply:
I saw `experimental` was taken out in the intro. Should it be taken out here as well?

Reply:
Yes, all the transpiler passes can run on local except the new one for PauliNetworks https://github.com/Qiskit/documentation/pull/2263

Reply:
Ok, I'll remove the experimental word. Thanks

Reply:
Fixed in https://github.com/Qiskit/documentation/pull/2273/commits/ab96f534a495708347f73a803bbd3c8ed2c8b073